src/components/__tests__/FormInviteFriends.cy.js

```diff
@@ -0,0 +1,175 @@
+            },
+          );
+        });
+      cy.dataCy('text-message')
+        .should('be.visible')
+        .and('have.css', 'font-size', '14px')
+        .and('have.css', 'font-weight', '400')
+        .and('have.color', black)
+        .then(($el) => {
+          const textContent = $el.text();
+          cy.stripHtmlTags(i18n.global.t('onboarding.textMessage')).then(
+            (text) => {
+              expect(textContent).to.contain(text);
+            },
+          );
+        });
+    });
+
+    it('allows to change the message language', () => {
+      cy.dataCy('invite-language-input').select('English');
+      cy.dataCy('title-message')
+        .should('be.visible')
+        .then(($el) => {
+          const textContent = $el.text();
+          cy.stripHtmlTags(i18n.global.t('onboarding.titleMessage', 'en')).then(
+            (text) => {
+              expect(textContent).to.contain(text);
+            },
+          );
+        });
+      cy.dataCy('text-message')
+        .should('be.visible')
+        .then(($el) => {
+          const textContent = $el.text();
+          cy.stripHtmlTags(i18n.global.t('onboarding.textMessage', 'en')).then(
+            (text) => {
+              expect(textContent).to.contain(text);
+            },
+          );
+        });
+      cy.dataCy('invite-language-input').select('Slovak');
+      cy.dataCy('title-message')
+        .should('be.visible')
+        .then(($el) => {
+          const textContent = $el.text();
+          cy.stripHtmlTags(i18n.global.t('onboarding.titleMessage', 'sk')).then(
+            (text) => {
+              expect(textContent).to.contain(text);
+            },
+          );
+        });
+      cy.dataCy('text-message')
+        .should('be.visible')
+        .then(($el) => {
+          const textContent = $el.text();
+          cy.stripHtmlTags(i18n.global.t('onboarding.textMessage', 'sk')).then(
+            (text) => {
+              expect(textContent).to.contain(text);
+            },
+          );
+        });
+      cy.dataCy('invite-language-input').select('Czech');
+      cy.dataCy('title-message')
+        .should('be.visible')
+        .then(($el) => {
+          const textContent = $el.text();
+          cy.stripHtmlTags(i18n.global.t('onboarding.titleMessage', 'cs')).then(
+            (text) => {
+              expect(textContent).to.contain(text);
+            },
+          );
+        });
+      cy.dataCy('text-message')
+        .should('be.visible')
+        .then(($el) => {
+          const textContent = $el.text();
+          cy.stripHtmlTags(i18n.global.t('onboarding.textMessage', 'cs')).then(
+            (text) => {
+              expect(textContent).to.contain(text);
+            },
+          );
+        });
+    });
+
+    it('renders columns side-by-side', () => {
+      cy.testElementPercentageWidth(cy.dataCy('column-1'), 50);
+      cy.testElementPercentageWidth(cy.dataCy('column-2'), 50);
+    });
+  });
+
```

Comment:
Missing test for testing `invite-email-addresses-input` text area widget validation rules?

Reply:
Added validation for `isFilled` rule and `isEmailList` rule.